README.md

```diff
@@ -6,10 +6,11 @@ This project tries to reproduce Ubuntu’s `command-not-found` for Homebrew user
 on OSX.
 
 On Ubuntu, when you try to use a command that doesn’t exist locally but is
-available through a package, Bash will suggest you a command to install it.
+available through a package, your shell will suggest you a command to install it.
+
 Using this script, you can replicate this feature on OSX:
 
-```
+```bash
```

Comment:
Please don’t add this, the code snippet is technically not in Bash (e.g. `$ when` include a prompt).